### DISCUSSION TITLE
How to shorten the content of area of one page?

### DISCUSSION THREAD

**Discussion by mendax1234** [squidfunk/mkdocs-material#6152] [Q&A] [answered]
I only want to shorten the content of the area of one page, what should l do? After reading the related topic in mkdocs material, l find that the provided method will applied to all of the pages on the site. 
Besides, how can l only shorten the area of the content, not including the toppings since I found that the example provided will extend or shorten everything.

Thanks!

**Comment by alexvoss** [comment]
Can you point me to the part of the documentation you were looking at, please? I am not sure I understand your question.

**Reply by mendax1234** [reply]
Okkkk. Sorry for causing you the trouble.
 https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#content-area-width This is the documentation part I am looking at.

**Reply by alexvoss** [reply, marked as answer]
One approach I can think of is to produce a [theme extension](https://squidfunk.github.io/mkdocs-material/customization/#extending-the-theme) and then [setting the page template](https://squidfunk.github.io/mkdocs-material/reference#setting-the-page-template) for the pages where you want a different width. The template should then define a CSS class you can match in your `extra.css` to produce the desired width only for those pages that need it. 

It may be more straightforward to do this with just CSS if there is something you can match that is already unique to the pages were you want to have a different width. If I understand CSS correctly, it would need to be an element or an attribute. It is not possible to match on content (such as the text in the top-level heading).

Think about whether there will be other things that may need to be adjusted for the pages you have in mind. If the answer is yes then it is probably worth investing the time to learn about extending themes.

**Reply by mendax1234** [reply]
Thank you very much!
